packages/wallets/src/features/cashier/modules/Transfer/components/TransferMessages/TransferMessages.tsx

```diff
@@ -30,7 +30,7 @@ const TransferMessages: React.FC = () => {
     return (
         <FadedAnimatedList className='wallets-transfer-messages'>
             {messages.map(({ action, message: { text, values }, type }) => {
-                const message = <Trans defaults={text} values={values} />;
+                const message = <Localize i18n_default_text={text} values={values} />;
```

Comment:
TODO: Verify this implementation for TransferMessages and text in this format are able to be updated to crowdin in the Transfer/Transactions translations card. Current implementation only removes usage of `Trans` and does not check the actual translation implementation of these messages